### DISCUSSION TITLE
Inviting @mayals to project

### DISCUSSION THREAD

**Discussion by elena** [djangonaut-space/program#14] [General]
Having a conversation with https://github.com/mayals and think she may be interested in the being part of this initiative. 

Just officially putting her name forward to start a discussion.

**Comment by RachellCalhoun** [comment]
Hi! Thanks putting May's name in! We appreciate the interest in the program, and as we prepare the next session we wanted to try to get all interested folks organized in one place. So if you or [@mayals ](https://github.com/mayals) would like to participate in a future session of the program please fill out this form https://forms.gle/4vNuqQwEK6o48WqHA and we'll reach out again before we start our next session. Hope to hear from you all soon :)
